doctr/datasets/classification/base.py

```diff
@@ -31,11 +32,12 @@ def synthesize_char_img(char: str, size: int = 32, font_family: Optional[str] =
 
     # Draw the character
     if font_family is None:
-        print(platform.system())
-        if platform.system() == 'Linux':
-            font = ImageFont.truetype("FreeMono.ttf", size)
-        else:
-            font = ImageFont.truetype("Arial.ttf", size)
+        try:
+            font = ImageFont.truetype("FreeMono.ttf" if platform.system() == "Linux" else "Arial.ttf", size)
+        except OSError:
+            font = ImageFont.load_default()
+            logging.warning("unable to load specific font families. Loading default PIL font,"
+                            "font size issues may be expected, you may want to specify the font_family in args")
```

Comment:
"font size issues may be expected. To prevent this, it is recommended to specify the value of 'font_family'."